src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package-lock.json

```diff
@@ -854,7 +767,8 @@
     "@types/selenium-webdriver": {
       "version": "3.0.15",
       "resolved": "https://registry.npmjs.org/@types/selenium-webdriver/-/selenium-webdriver-3.0.15.tgz",
-      "integrity": "sha512-5nh8/K2u9p4bk95GGCJB7KBvewaB0TUziZ9DTr+mR2I6RoO4OJVqx7rxK83hs2J1tomwtCGkhiW+Dy8EUnfB+Q=="
+      "integrity": "sha512-5nh8/K2u9p4bk95GGCJB7KBvewaB0TUziZ9DTr+mR2I6RoO4OJVqx7rxK83hs2J1tomwtCGkhiW+Dy8EUnfB+Q==",
+      "optional": true
```

Comment:
Not sure why this is getting added. I'm using npm 6.9.0 and node 10.16.0. It might be that we used an older node version before or that the format changed with a later version of node.

In any case, we should migrate it to this format as angular8 will require node10 (current LTS) and doesn't support previous versions.